add-on/src/lib/ipfs-companion.js

```diff
@@ -158,7 +158,11 @@ module.exports = async function init () {
     const info = {
       ipfsNodeType: state.ipfsNodeType,
       peerCount: state.peerCount,
-      repoStats: state.repoStats,
+      // Convert big.js numbers into strings before sending.
+      // Chrome uses JSON.stringify to send objects over port.postMessage whereas
+      // Firefox uses structured clone. It means that on the other side FF gets
+      // a weird object (not a big.js number object) unless we stringify first.
+      repoStats: JSON.parse(JSON.stringify(state.repoStats)),
```

Comment:
Yup, this fixed the problem mentioned in https://github.com/ipfs-shipyard/ipfs-companion/pull/389#issuecomment-367136706, thanks! :+1: